app/models/competition_tab.rb

```diff
@@ -34,6 +34,16 @@ def slug
     competition.tabs.where("display_order > ?", display_order).update_all("display_order = display_order - 1")
   end
 
+  validate :verify_if_full_urls, on: :update
+  private def verify_if_full_urls
```

Comment:
Also, can we have some basic tests for this feature? RegExps are always difficult to do correctly without trying/testing.

Reply:
Done